pkg/eth/cid_retriever.go

```diff
@@ -316,12 +316,12 @@ func (ecr *CIDRetriever) RetrieveFilteredGQLLogs(tx *sqlx.Tx, rctFilter ReceiptF
 	id := 1
 	pgStr := `SELECT eth.log_cids.leaf_cid, eth.log_cids.index, eth.log_cids.receipt_id,  
        			eth.log_cids.address, eth.log_cids.topic0, eth.log_cids.topic1, eth.log_cids.topic2, eth.log_cids.topic3, 
-       			eth.log_cids.log_data, eth.transaction_cids.tx_hash, data, eth.receipt_cids.cid, eth.receipt_cids.post_status
+       			eth.log_cids.log_data, eth.transaction_cids.tx_hash, data, eth.receipt_cids.leaf_cid as cid, eth.receipt_cids.post_status
 				FROM eth.log_cids, eth.receipt_cids, eth.transaction_cids, eth.header_cids, public.blocks
 				WHERE eth.log_cids.receipt_id = receipt_cids.id
 				AND receipt_cids.tx_id = transaction_cids.id
  				AND transaction_cids.header_id = header_cids.id
- 				AND log_cids.leaf_mh_key = blocks.key AND header_cids.block_hash = $1`
+ 				AND receipt_cids.leaf_mh_key = blocks.key AND header_cids.block_hash = $1`
```

Comment:
Brain is foggy and I'm not reasoning about this well atm, why are we changing this condition from `log_cids.leaf_mh_key = blocks.key` to  `receipt_cids.leaf_mh_key = blocks.key`?

Reply:
Reverted.